.storybook/blocks/PropertiesTable.jsx

```diff
@@ -0,0 +1,205 @@
+                        <path d="M18 2a16 16 0 1 0 16 16A16 16 0 0 0 18 2Zm10.666 9.08L16.018 27.341a1.208 1.208 0 0 1-.875.461c-.024.002-.05.002-.073.002a1.2 1.2 0 0 1-.85-.351l-7.784-7.795a1.2 1.2 0 0 1 0-1.698l1.326-1.325a1.201 1.201 0 0 1 1.695 0l5.346 5.347L25.314 8.473A1.203 1.203 0 0 1 27 8.263l1.455 1.133a1.205 1.205 0 0 1 .211 1.684Z"></path>
+                    ) : variant === "negative" ? (
+                        <path d="M17.127 2.579.4 32.512A1 1 0 0 0 1.272 34h33.456a1 1 0 0 0 .872-1.488L18.873 2.579a1 1 0 0 0-1.746 0ZM20 29.5a.5.5 0 0 1-.5.5h-3a.5.5 0 0 1-.5-.5v-3a.5.5 0 0 1 .5-.5h3a.5.5 0 0 1 .5.5Zm0-6a.5.5 0 0 1-.5.5h-3a.5.5 0 0 1-.5-.5v-12a.5.5 0 0 1 .5-.5h3a.5.5 0 0 1 .5.5Z"></path>
+                    ) : variant === "info" ? (
+                        <path d="M18 2a16 16 0 1 0 16 16A16 16 0 0 0 18 2Zm-.3 4.3a2.718 2.718 0 0 1 2.864 2.824 2.664 2.664 0 0 1-2.864 2.863 2.705 2.705 0 0 1-2.864-2.864A2.717 2.717 0 0 1 17.7 6.3ZM22 27a1 1 0 0 1-1 1h-6a1 1 0 0 1-1-1v-2a1 1 0 0 1 1-1h1v-6h-1a1 1 0 0 1-1-1v-2a1 1 0 0 1 1-1h4a1 1 0 0 1 1 1v9h1a1 1 0 0 1 1 1Z"></path>
+                    ) : null}
+                </svg>
+                <div className="spectrum-Toast-body">
+                    <div className="spectrum-Toast-content">{children}</div>
+                </div>
+                <div className="spectrum-Toast-buttons">
+                    <button
+                        aria-label="close"
+                        className="spectrum-CloseButton spectrum-CloseButton--sizeM spectrum-CloseButton--staticWhite"
+                        label="Close"
+                    >
+                        <svg
+                            className="spectrum-Icon spectrum-UIIcon-Cross100 spectrum-Icon--medium spectrum-CloseButton-UIIcon"
+                            focusable="false"
+                            aria-hidden="true"
+                            role="img"
+                            xmlns="http://www.w3.org/2000/svg"
+                            viewBox="0 0 8 8"
+                        >
+                            <path d="m5.238 4 2.456-2.457A.875.875 0 1 0 6.456.306L4 2.763 1.543.306A.875.875 0 0 0 .306 1.544L2.763 4 .306 6.457a.875.875 0 1 0 1.238 1.237L4 5.237l2.456 2.457a.875.875 0 1 0 1.238-1.237z"></path>
+                        </svg>
+                    </button>
+                </div>
+            </div>
+		</Container>
+	);
+};
+
+/**
+ * Write the provided text to the clipboard using the Clipboard API and
+ * show a toast message to indicate the operation was successful or not.
+ *
+ * @param {string} text
+ */
+export function writeTextToClipboard(text) {
+	function updateMessage(toast, content) {
+		const container = toast.querySelector(".spectrum-Toast-content");
+		if (container) container.innerHTML = content;
+		else return;
+	}
+
+	function toggleToast(toast, open = true) {
+		toast.classList.toggle("is-open", open);
+	}
+
+	navigator.clipboard.writeText(text).then(
+		() => {
```

Comment:
There's something going on that is causing this to be called multiple times on click. After clicking one of the mod properties, I get 5 console logs with the same info, instead of one.